### PR TITLE
fix(namespace): add the field namespaceOverride back.

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#6527

#### What this PR does / why we need it:

This field is used to override the Longhorn namespace when Longhorn is a sub-chart such as a sub-chart of the Harvester.

### CODE REVIEW

File: chart/README.md [hunk 1]
```diff
@@ -243,6 +243,7 @@ For more details, see the [ocp-readme](https://github.com/longhorn/longhorn/blob
 | enableGoCoverDir | `false` | Setting that allows Longhorn to generate code coverage profiles. |
 | enablePSP | `false` | Setting that allows you to enable pod security policies (PSPs) that allow privileged Longhorn pods to start. This setting applies only to clusters running Kubernetes 1.25 and earlier, and with the built-in Pod Security admission controller enabled. |
 | metrics.serviceMonitor.enabled | `false` | Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components. |
+| namespaceOverride | `""` | Specify override namespace, specifically this is useful for using longhorn as sub-chart and its release namespace is not the `longhorn-system` such as Harvester. |
```
Comment (innobead) [inline chart/README.md:246]: No need to specially mention Harvester.

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one feedback.

**Comment by innobead** [comment]
@mergify backport v1.6.x

File: chart/README.md (hunk 1 above)
Comment (mantissahz) [reply chart/README.md]: Modified. Thanks.

**Review by innobead — APPROVED** [review]
Please update the website doc.

**Pull request merged**